bitnami/external-dns/values.yaml

```diff
@@ -26,6 +26,15 @@ global:
       ## @param global.compatibility.openshift.adaptSecurityContext Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
       ##
       adaptSecurityContext: auto
+    ## Compatibility adaptations for Traefik
+    ##
+    traefik:
+      ## @param global.compatibility.traefik.disableNew Disable listeners on Resources under traefik.io
+      ##
+      disableNew: false
+      ## @param global.compatibility.traefik.disableLegacy Disable listeners on Resources under traefik.containo.us
+      ##
+      disableLegacy: false
```

Comment:
I think it is not necessary to define these new values under global, which is a [reserved section in values.yaml](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values).

What if you set these new values as `.Values.traefik.disableLegacy` and `.Values.traefik.disableNew`?